src/System Application/App/Email/src/Email/Sent/SentEmails.Page.al

```diff
@@ -211,7 +213,7 @@ page 8883 "Sent Emails"
         EmailConnector.ShowAccountInformation(Rec."Account Id");
     end;
 
-    internal procedure SetNewerThan(NewDate: DateTime)
+    procedure SetNewerThan(NewDate: DateTime)
```

Comment:
Why do you need this method to be public?

Reply:
also, page 8888 "Sent Emails List Part" has these methods public and documented. I would think we need to align these two pages unless there is a good reason not to.

Reply:
@grobyns The page is public and can be called directly via .Run/RunModal. I wanted to create the option of directly setting the date filter in this process. Setting a filter on a page is not a sensitive functionality. Therefore, I would like to allow the access.

Reply:
but then why not also SetRelatedRecord? Why only SetNewerThanDate? 

As I've also mentioned, these methods are already public on the list part so I think we should align the pages so they have the same public methods (and with documentation)

Reply:
@grobyns Yes, I agree. I have aligned the function comments and set the procedure "SetRelatedRecord" to public. Object update is sent.